SignalMessaging/views/OWSAlerts.swift

```diff
@@ -65,4 +65,32 @@ import Foundation
         return action
     }
 
+    @objc
+    public class func showIOSUpgradeNagIfNecessary() {
+        // Only show the nag to iOS 8 users.
+        if #available(iOS 9.0, *) {
+            return
+        }
+
+        // Don't show the nag to users who have just launched
+        // the app for the first time.
+        guard let _ = AppVersion.instance().lastAppVersion else {
```

Comment:
style nit: 

Rather than using both implicit-unwrap and ignored-return, I think it would be clearer to just compare to nil

```
guard AppVersion.instance().lastAppVersion != nil else {
```

Reply:
Done.